docs/k8s/getting-started-gwapi.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Getting Started - Kubernetes Gateway API
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+import K8sIngressInstallExample from "/examples/k8s/install-ingress.mdx";
+import K8sGatewayAPIInstallExample from "/examples/k8s/install-gateway-api.mdx";
+
+import K8sIngressModifyHeadersExample from "/examples/k8s/http-request-headers.mdx";
+import K8sGatewayModifyHeadersExample from "/examples/k8s/http-request-headers-gateway.mdx";
+
+import K8sIngressRedirectExample from "/examples/k8s/http-request-redirect.mdx";
+import K8sGatewayRedirectExample from "/examples/k8s/http-request-redirect-gateway.mdx";
+
+import StaticDomainKubernetesExample from "/examples/k8s/http-static-domain.mdx";
+import StaticDomainKubernetesGatewayExample from "/examples/k8s/http-static-domain-gateway.mdx";
+
+import StaticDomainKubernetesPathExample from "/examples/k8s/http-static-domain-path.mdx";
+import StaticDomainKubernetesGatewayPathExample from "/examples/k8s/http-static-domain-gateway-path.mdx";
+
+# Kubernetes Gateway API
+
+## Overview
+
+The Kubernetes Operator allows access to k8s Services using the new [Gateway API](https://gateway-api.sigs.k8s.io/).
```

Comment:
Let's use Kubernetes everywhere instead of `k8s` and `K8s` unless it's in a URL or copied / quoted text.

Reply:
```suggestion
The Kubernetes Operator allows access to Kubernetes Services using the new [Gateway API](https://gateway-api.sigs.k8s.io/).
```